src/common/dump.h

```diff
@@ -30,6 +30,10 @@ enum {
     DUMP_FLASH_SIZE = 0x00100000,
 };
 
+// DUMP constants for error message
+#define DUMP_MSG_TITLE_MAX_LEN 20
+#define DUMP_MSG_MAX_LEN       107
+static const uint32_t DUMP_MSG_SIZE = DUMP_MSG_TITLE_MAX_LEN + DUMP_MSG_TITLE_MAX_LEN + 1;
```

Comment:
note: DUMP_MSG_SIZE is never used anywhere and is wrong anyway as sizeof(unsigned short flag) is probably 2.

Reply:
That was a mistake, the flag should be uint8_t. Fixed in `7c21391`